### PR TITLE
Update migrate-mailboxes-across-tenants.md

### PR DESCRIPTION
**Pull request by tweekerz**
Reviewed with PMs on this

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/migrate-mailboxes-across-tenants.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ localization_priority: Priority
 # How to migrate mailboxes from one Microsoft 365 or Office 365 organization to another
 
    > [!NOTE]
-   > At this time, a built-in tenant to tenant migration option is not available.
+   > At this time, we have a Public Preview of the native Cross-tenant mailbox migration located at <https://aka.ms/CrossTenantMailboxMigration>
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/migrate-mailboxes-across-tenants.md:29]:
```suggestion
   > At this time, we have a Public Preview of the native Cross-tenant mailbox migration located at <https://aka.ms/CrossTenantMailboxMigration>.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**